addons/project_mrp/views/mrp_production_views.xml

```diff
@@ -23,8 +23,8 @@
                 }</attribute>
             </field>
             <xpath expr="//page[@name='miscellaneous']//field[@name='date_deadline']" position="after">
-                <t groups="stock.group_stock_manager">
-                    <field name="project_id" groups="project.group_project_user"/>
+                <t groups="project.group_project_user">
+                    <field name="project_id" />
```

Comment:
I would keep the groups defined on the field to be sure we don't forget to keep it in master when we will remove the t tag. 🙂 
```suggestion
                    <field name="project_id" groups="project.group_project_user"/>
```